docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,48 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+[[openshift-docker-registry-intro]]
+== OpenShift Docker Registry
+
+OpenShift provides an integrated docker registry which can be used for local development as well.
+
+This helps the local development as the images present in registry can be used for applications running in OpenShift. Refer to the link:../using/accessing-the-openshift-docker-registry{outfilesuffix}#staging-app-with-openshift-registry[staging your application with the OpenShift registry] to know more.
```

Comment:
...the registry...
Refer to the 'S'taging your application with the OpenShift registry section...